lib/Support/Support.cpp

```diff
@@ -99,4 +99,22 @@ const std::string strFormat(const char *format, ...) {
   va_end(vaArgs);
   return std::string(str.data(), len);
 }
+
+std::string legalizeName(llvm::StringRef name) {
```

Comment:
can you add quick test for this

Reply:
I modified the Caffe2ImporterTest.cpp test to verify this case. Do you mean we need to write a separate test for the function only?

Reply:
yup, to cover those cases when it does not start with alpha, when it's empty etc.

Reply:
Tests added.